lib/walker.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+function init_syntax(){
+   /*
+      Initializes an object for iterating over esprima nodes.
+
+      The `syntax` variable and the for loop to modify it are taken directly 
+      from `istanbul` npm module (istanbul/lib/instrumenter.js).
+
+      @return: an object with a key for each esprima node type
+         the value of each key is an object with `type` equal to the key and 
+         `children` which allows us to extract the relevant sub-nodes from each 
+         esprima node
+   */
+
+   var syntax = {
+      // keep in sync with estraverse's VisitorKeys
+      AssignmentExpression: ['left', 'right'],
```

Comment:
I'm a little confused by this comment -- do you mean I don't have to make *all* the node types constants somewhere, or that I should change this syntax var somehow?